### DISCUSSION TITLE
What is Vercel.com made of?

### DISCUSSION THREAD

**Discussion by qnxdev** [vercel/vercel#4319] [General] [answered]
What is Vercel.com made of? Liked it anyways.

**Comment by styfle** [comment, marked as answer]
Hi @qnxdev 

If you mean, "who" is Vercel made of? You can see our team and our investors on the about page: https://vercel.com/about

If you mean, "what technologies" is Vercel made of? We host Vercel.com on Vercel using Next.js: https://nextjs.org
